climate-monitoring-client/src/main/java/climatemonitoring/ProxyImpl.java

```diff
@@ -14,6 +14,8 @@
 import java.io.ObjectOutputStream;
 import java.net.Socket;
 
+import javax.xml.crypto.Data;
+
```

Comment:
What is this is import??? You should remove it

Reply:
lol